sys/include/net/gnrc/mac/types.h

```diff
@@ -40,12 +37,151 @@ extern "C" {
  * @brief definition for device transmission feedback types
  */
 typedef enum {
-    TX_FEEDBACK_UNDEF = 0,    /* Transmission just start, no Tx feedback yet */
-    TX_FEEDBACK_SUCCESS,      /* Transmission succeeded */
-    TX_FEEDBACK_NOACK,        /* No ACK for the transmitted packet */
-    TX_FEEDBACK_BUSY          /* found medium busy when doing transmission */
+    TX_FEEDBACK_UNDEF = 0,    /**< Transmission just start, no Tx feedback yet */
+    TX_FEEDBACK_SUCCESS,      /**< Transmission succeeded */
+    TX_FEEDBACK_NOACK,        /**< No ACK for the transmitted packet */
+    TX_FEEDBACK_BUSY          /**< found medium busy when doing transmission */
 } gnrc_mac_tx_feedback_t;
 
+/**
+ * @brief Static initializer for gnrc_mac_tx_feedback_t.
+ */
+#define GNRC_MAC_TX_FEEDBACK_INIT { TX_FEEDBACK_UNDEF }
+
+#if ((GNRC_MAC_RX_QUEUE_SIZE != 0) || (GNRC_MAC_DISPATCH_BUFFER_SIZE != 0))
```

Comment:
Yes, but that is only the case (at least if you don't change stuff to the buildsystem ;-)) when the doxygen documentation is rendered (otherwise the documentation in the `#if` branch would be excluded from http://doc.riot-os.org ;-) because doxygen is considering the defined macros). Try `make doc` in the RIOTBASE directory to build and test the doxygen locally (you need to have doxygen installed for that of course).